### PR TITLE
FYLE-2ve2ey-1: Tfc revamp - Custom inputs changes 

### PR DESCRIPTION
**Pull request by Dimple16**

### CODE REVIEW

File: src/app/core/services/custom-fields.service.ts [hunk 1]
```diff
@@ -34,19 +34,19 @@ export class CustomFieldsService {
     let property = {
       id: customInput.id,
       prefix,
-      name: customInput[prefix + 'name'],
+      name: customInput[prefix + 'field_name'],
```
Comment (madhavmansuriya40) [inline src/app/core/services/custom-fields.service.ts:37]: at some places, custom fields are with prefix and at someplace they are direct
please test it in all cases

**Review by madhavmansuriya40 — APPROVED** [review]

File: src/app/core/services/custom-inputs.service.ts [hunk 2]
```diff
@@ -15,15 +16,24 @@ export class CustomInputsService {
...
     private decimalPipe: DecimalPipe,
-    private datePipe: DatePipe
+    private datePipe: DatePipe,
+    private authService: AuthService
   ) { }
 
   @Cacheable({
     cacheBusterObserver: customInputssCacheBuster$
   })
   getAll(active: boolean) {
```
Comment (iamtarun1993) [inline src/app/core/services/custom-inputs.service.ts:26]: return type

File: src/app/core/services/custom-fields.service.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/core/services/custom-fields.service.ts:37]: yes, this is done

**Review by abhishek1234321 — CHANGES_REQUESTED** [review]

File: src/app/core/services/custom-inputs.service.ts [hunk 3]
```diff
@@ -15,15 +17,24 @@ export class CustomInputsService {
...
+    private authService: AuthService
   ) { }
 
   @Cacheable({
     cacheBusterObserver: customInputssCacheBuster$
   })
-  getAll(active: boolean) {
-    return this.apiService.get('/custom_inputs/custom_properties', { params: { active } }).pipe(
-      shareReplay(1)
+  getAll(active: boolean): Observable<ExpenseField[]> {
```
[the remark this answers is not in the document]
Comment (Dimple16) [reply src/app/core/services/custom-inputs.service.ts:27]: passing active parameter

File: src/app/core/services/custom-fields.service.ts (hunk 1 above)
Comment (Dimple16) [reply src/app/core/services/custom-fields.service.ts:37]: `setProperty` is called only from `standardizeCustomFields`, `standardizeCustomFields` this method is used in expense forms and view trip/team trip pages

**Review by abhishek1234321 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]

File: src/app/core/services/custom-inputs.service.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (Julias0) [reply src/app/core/services/custom-inputs.service.ts:27]: `getAll({ active: boolean } = { active: false })`

**Pull request closed without merging**